### PR TITLE
2-improved Python

### PR DESCRIPTION
**Pull request by YaroslavKSE**
Create function sum(*args) returning sum of arguments

### CODE REVIEW

**Review by ykhadnevich — COMMENTED** [review]
Все найс

**Comment by YaroslavKSE** [comment]
Okay, launching to main

**Comment by tshemsedinov** [comment]
@YaroslavKSE `landing` not `launching`

File: 0-buggy.py [hunk 1]
```diff
@@ -1,4 +1,8 @@
-first_num = 2
-secord_num = 3
-sum = first_num + second_num
-print(sum)
+def sum(*args):
+    total_sum = 0
+    for number in args:
+        total_sum += number
+    return total_sum
+
```
Comment (tshemsedinov) [inline 0-buggy.py:6]: 1 line is enough

**Pull request merged**